source/releasenotes/changes.rst

```diff
@@ -13,6 +13,336 @@
    specific language governing permissions and limitations
    under the License.
 
+Changes in |release| since 4.17.0.0
+===================================
+
+Apache CloudStack uses GitHub https://github.com/apache/cloudstack/milestone/25?closed=1
+to track its issues.
+
+.. cssclass:: table-striped table-bordered table-hover
+
+
++-------------------------+----------+---------------+----------+------------------------------------------------------------+
+| Version                 | Github   | Type          | Priority | Description                                                |
```

Comment:
We can remove the type and priority columns if they're empty (maybe fix the py script that generates the table)?

Reply:
Tried but it messes with formatting. Need to spend more time. For later